### PR TITLE
bitcoind: add page

### PR DESCRIPTION
**Pull request by fazlearefin**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by fazlearefin** [comment]
Note that the `=` is necessary for the commands to work properly. Omitting `=` won't work.

File: pages/common/bitcoind.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+- Start the Bitcoin Core daemon (in the foreground):
+
+`bitcoind`
+
+- Start the Bitcoin Core daemon in the background (use `bitcoin-cli stop` to stop):
+
+`bitcoind -daemon`
+
+- Start Bitcoin Core daemon on a specific network:
```
Comment (spageektti) [inline pages/common/bitcoind.md:15]:
```suggestion
- Start the Bitcoin Core daemon on a specific network:
```

File: pages/common/bitcoind.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+`bitcoind -chain={{main|test|signet|regtest}}`
+
+- Start Bitcoin Core daemon using specific config file and data directory:
```
Comment (spageektti) [inline pages/common/bitcoind.md:19]:
```suggestion
- Start the Bitcoin Core daemon using specific config file and data directory:
```

File: pages/common/bitcoind.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# bitcoind
+
+> Bitcoin Core daemon.
+> Uses the configuration defined in `bitcoin.conf`.
+> More information: <https://manned.org/man/bitcoind>.
```
Comment (spageektti) [inline pages/common/bitcoind.md:5]:
```suggestion
> More information: <https://manned.org/bitcoind>.
```

**Review by spageektti — APPROVED** [review]

**Pull request merged**